airbyte-integrations/connectors/source-apple-search-ads/manifest.yaml

```
@@ -415,7 +415,7 @@ definitions:
             selector: >-
               { "orderBy": [ { "field": "countryOrRegion", "sortOrder":
               "ASCENDING" } ] }
-            timeZone: UTC
+            timeZone: "{{ config.time_zone }}"
```

```suggestion
            timeZone: "{{ config.time_zone or "UTC" }}"
```
This way won't introduce a breaking change to existing syncs.